web/src/pages/InvoicePage/InvoicePage.tsx

```diff
@@ -0,0 +1,35 @@
+// import { Link, routes } from '@redwoodjs/router'
+import { Download, FileSearch, FileText } from 'lucide-react'
+import { Button } from 'web/src/components/ui/button'
+
+import { Metadata } from '@redwoodjs/web'
+
+const InvoicePage = () => {
+  return (
+    <>
+      <Metadata title="Invoice" description="Invoice page" />
+      <div className="relative -left-4 -top-4 min-h-[120vh] w-screen bg-white">
```

Comment:
Are these class names intentional?

Reply:
Made some changes to address this question. ☺️